inst/sql/sql_server/table_cdm_table.sql

```diff
@@ -0,0 +1,29 @@
+
+/*********
+TABLE LEVEL check:
+CDM_TABLE - verify the table exists
+
+Parameters used in this template:
+cdmDatabaseSchema = @cdmDatabaseSchema
+cdmTableName = @cdmTableName
+
+**********/
+
+
+SELECT num_violated_rows, CASE WHEN denominator.num_rows = 0 THEN 0 ELSE 1.0*num_violated_rows/denominator.num_rows END AS pct_violated_rows, 
+  denominator.num_rows as num_denominator_rows
+FROM
+(
+  select num_violated_rows from
+  (
+    select
+      case when count_big(*) = 0 then 0
+      else 0
+    end as num_violated_rows
```

Comment:
I see this is the same in [the `cdmField` check](https://github.com/OHDSI/DataQualityDashboard/blob/main/inst/sql/sql_server/field_cdm_field.sql), but I am wondering why this is necessary. Maybe @alondhe or @clairblacketer can comment.

My suggestion to simplify (again same applies to cdmField check):
```suggestion
      0 as num_violated_rows
```

Reply:
If others agree, I will simplify it according to your suggestion.

Reply:
Agreed, I will update the commit.